<launch>
        <rosparam command="load" file="$(find robolink)/params/joint_0.yaml" ns="joint_0" />
        <rosparam command="load" file="$(find robolink)/params/joint_1.yaml" ns="joint_1" />
		<rosparam command="load" file="$(find robolink)/params/joint_2.yaml" ns="joint_2" />
		<rosparam command="load" file="$(find robolink)/params/joint_3.yaml" ns="joint_3" />
		<rosparam command="load" file="$(find robolink)/params/joint_4.yaml" ns="joint_4" />
              
        <node name="robolink" pkg="EposManager" type="EposManager"  output="screen" args="joint_0 joint_1 joint_2 joint_3 joint_4" >
                <param name="Protocol" value="USB" />
                <param name="Port" value="USB0" />
                <param name="Publish_Rate" value="10" />
        </node>
        
        
		<node name="joy_node" pkg="joy" type="joy_node" >
        		<param name="deadzone" value="0.15" />
        </node>     
        
        <node name="RobolinkDriver" pkg="robolink" type="RobolinkDriver.py" output="screen" >
        </node>
        
</launch>
